package.json

```diff
@@ -66,11 +66,11 @@
     "eslint-plugin-standard": "^4.0.0",
     "eslint-plugin-vue": "^5.2.3",
     "flush-promises": "^1.0.2",
-    "jest": "^24.9.0",
+    "jest": "^25.1.0",
     "jest-vue-preprocessor": "^1.7.1",
     "node-sass": "^4.13.1",
     "postcss": "^7.0.27",
-    "postcss-cli": "^6.1.3",
```

Comment:
Are these updates to dependencies intentional? Just maintenance?

Reply:
Security updates (npm audit)